sdk/internal/perf/utils.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package perf
+
+import "fmt"
+
+// Helper function to sum a slice of integers
+func sumInts(ints []int) int {
+	var ret int
+	for _, i := range ints {
+		ret += i
+	}
+	return ret
+}
+
+// convert an integer to a string, left padding with zeros
+func leftPad(i int) string {
+	if i >= 100 {
+		return fmt.Sprintf("%d", i)
+	} else if i >= 10 {
+		return fmt.Sprintf("0%d", i)
+	} else if i > 0 {
+		return fmt.Sprintf("00%d", i)
+	}
+	return "000"
+}
+
+// Convert an integer to a string with comma separation in the format 1234 -> 1,234
+func commaIze(i int) string {
+	if i < 1000 {
+		return fmt.Sprintf("%d", i)
+	}
+
+	copy := i
+	ret := leftPad(copy % 1000)
+	copy /= 1000
+	for copy >= 1000 {
+		temp := leftPad(copy % 1000)
+		ret = fmt.Sprintf("%s,%s", temp, ret)
+
+		copy /= 1000
+	}
+
+	if copy == 0 {
+		return ret
+	}
+
+	return fmt.Sprintf("%d,%s", copy, ret)
+}
+
+func min(a, b int) int {
```

Comment:
Could you use this instead?

https://pkg.go.dev/math#Min

Reply:
Unfortunately that Min is for floats, after generics hopefully that will be fixed to be a generic type.